e2e-tests/local-testnet-debug.sh

```diff
@@ -38,8 +38,8 @@ PROVIDER_COMETMOCK_ADDR=tcp://$NODE_IP:22331
 CONSUMER_COMETMOCK_ADDR=tcp://$NODE_IP:22332
 
 # Clean start
-pkill -f interchain-security-pd &> /dev/null || true
-pkill -f cometmock &> /dev/null || true
+pkill -f ^interchain-security-pd &> /dev/null || true
+pkill -f ^cometmock &> /dev/null || true
 sleep 1
```

Comment:
nit: I guess that's here to guarantee the processes have been killed. If this is a real concern, you could test in a loop whether the processes have been killed and wait for 1 second every time.